gridscout

```diff
@@ -12,11 +12,16 @@ import sys
 from   schema import Schema, SchemaError
 import time
 
+import fcntl, termios, struct # Used in determining screen size
```

Comment:
These are on a separate line by themselves. I'd like to see if we really need them all before we mingle them in with everything else.